### PR TITLE
doc: edit recipes for clarity and grammar

### PR DESCRIPTION
**Pull request by kgriffs**

### CODE REVIEW

**Review by vytas7 — APPROVED** [review]
:+1:

File: docs/user/recipes/request-id.rst [hunk 1]
```diff
@@ -45,9 +47,12 @@ create a unique ID and set that in the thread local:
         def process_request(self, req, resp):
             ctx.request_id = str(uuid4())
 
+        # It may also be helpful to include the ID in the response
+        def process_response(self, req, resp, resource, req_succeeded):
+            resp.set_header('Request-ID', ctx.request_id)
```
Comment (CaselIT) [inline docs/user/recipes/request-id.rst:52]: Shouldn't we prefix with `X-` this custom headers? so `X-Request-ID`

File: docs/user/recipes/request-id.rst [hunk 2]
```diff
@@ -61,13 +66,33 @@ Alternatively if all of your application logic has access to the
...
             # Using Falcon 2.0 syntax
             req.context.request_id = request_id
 
             # Or if your logger has built-in support for contexts
             req.context.log = structlog.get_logger(request_id=request_id)
 
-Logging with the `thread-local`_ context can be done like:
+        # It may also be helpful to include the ID in the response
+        def process_response(self, req, resp, resource, req_succeeded):
+            resp.set_header('Request-ID', req.context.request_id)
```
Comment (CaselIT) [inline docs/user/recipes/request-id.rst:78]: same here

**Review by CaselIT — DISMISSED** [review]
just some nitpicks. I'm fine either way.

**Review by vytas7 — APPROVED** [review]

**Review by CaselIT — APPROVED** [review]

**Pull request merged**